src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.macios.cs

```diff
@@ -26,15 +26,15 @@ public partial class Snackbar
 	/// <summary>
 	/// Show Snackbar
 	/// </summary>
-	private partial async Task ShowNative(CancellationToken token)
+	private partial async Task ShowPlatform(CancellationToken token)
 	{
-		await DismissNative(token);
+		await DismissPlatform(token);
 		token.ThrowIfCancellationRequested();
 
 		var cornerRadius = GetCornerRadius(VisualOptions.CornerRadius);
 
-		var padding = GetMaximum(cornerRadius.X, cornerRadius.Y, cornerRadius.Width, cornerRadius.Height) + ToastView.DefaultPadding;
-		NativeSnackbar = new SnackbarView(Text,
+		var padding = GetMaximum(cornerRadius.X, cornerRadius.Y, cornerRadius.Width, cornerRadius.Height) + PlatformToast.DefaultPadding;
```

Comment:
I see here that we use the `DefaultPadding`, but what happens if the user changes it?

Reply:
The user will get different padding. 
It is a public setting.

Reply:
Default padding is a read-only and we don't set it inside the `PlatformToast`[ ctor](https://github.com/CommunityToolkit/Maui/blob/eb2606b34d56b96e8cb71eb0dd9f923ffa2d4e3a/src/CommunityToolkit.Maui.Core/Views/Toast/PlatformToast.macios.cs#L35-L50). So even if the user sets a padding on ctor, the DefaultPadding will be the same

Reply:
Ah, you are right. I need a cup of coffee. I thought it is in Defaults.shared.cs.
I don't remember why it is a public read only. It should be private constant. We need this property to look similar to Android Snackbar.

Reply:
so to summarize, it is used as a default padding. so if the user sets the corner radius equal to 0, we still have some padding equal to 10. but it was not correct to set it here. Now set default padding internally